sdk/schemaregistry/Microsoft.Azure.Data.SchemaRegistry.ApacheAvro/perf/Scenarios/Serialize.cs

```diff
@@ -0,0 +1,40 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Collections.Generic;
+using System.Threading;
+using System.Threading.Tasks;
+using Azure.Test.Perf;
+using TestSchema;
+
+namespace Microsoft.Azure.Data.SchemaRegistry.ApacheAvro.Perf.Scenarios
+{
+    public class Serialize : SchemaRegistryAvroSerializerPerfTestBase
+    {
+        private readonly List<Employee> _employees = new();
+
+        public Serialize(SizeCountOptions options) : base(options)
+        {
+            for (int i = 0; i < options.Count; i++)
```

Comment:
This is probably best done as part of `GlobalSetup` and used as a static set across parallel instances.